commitlint.config.ts

```diff
@@ -47,11 +47,15 @@ module.exports = {
         "too-many-spaces": [RuleConfigSeverity.Error, "always"],
         "commit-hash-alone": [RuleConfigSeverity.Error, "always"],
         "title-uppercase": [RuleConfigSeverity.Error, "always"],
+        "default-revert-message": [RuleConfigSeverity.Error, "never"],
```

Comment:
@realmarv when I raised that we should use 'never' here instead of always, I also meant that the plugin should not ignore this argument, it should act on it (e.g. imagine if we contribute this rule at some point to commitlint repo and people want to use it with 'always' argument so that their devs can't edit the revert msg? I know it would be stupid but the point of this arg is to invert the logic)

Reply:
If someone edits the revert message they might edit it in a way that we can't even catch it's a revert message. We can check for revert keyword in the header and body to make sure if the commit is a revert but the rule's logic will change a bit.

Reply:
Done